options.html

```diff
@@ -1,148 +1,172 @@
- <html>
-<head>
-	<link rel="stylesheet" type="text/css" href="css/options.css" />
-</head>
-	<body>
-		<div id="rikaikun_options">
-<!--			<div id="tabs">
-				<div id="general" class="tab selected-tab">General</div><div id="keyboard" class="tab active-tab">Keyboard</div><div id="worddict" class="tab">Word Dictionary</div><div id="kanjidict" class="tab active-tab">Kanji Dictionary</div><div id="namedict" class="tab">Names Dictionary</div><div id="clipboard" class="tab active-tab">Clipboard</div>
-			</div> -->
-			<form id="optform" name="optform">
-				<div id="options">
-					<div id="gencon" class="tabContent">
-					<h1>General</h1>
-						<p>Popup color:
-							<select class="inputfield" id="popupcolor" name="popupcolor">
-								<option value="blue">Blue</option>
-								<option value="lightblue">Light Blue</option>
-								<option value="black">Black</option>
-								<option value="yellow">Yellow</option>
-							</select>
-							<br />

```

Comment:
why did you delete the space after the br tags everywhere?

your editor is out of control.

This file has no functional changes at all does it? it's all just reformat?

General, rule is to not combine refacotring with functional changes since it makes it hard to separate what is important to which.